utils/auth/function.ts

```
@@ -213,7 +213,7 @@ export const checkProfileStatus = async (userId: string | undefined) => {
     }
 
     // Check if profile exists and has required fields
-    return data && data.user_id && data.username; // adjust based on your required fields
+    return data && data.id && data.username; // adjust based on your required fields
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)